tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -19,6 +19,15 @@ datasets:
     dataset_meta: mscoco_mask_rcnn_short_80.json
     data_source: val2017
 
+
+  - name: mrlEyes_2018_01
+    annotation_conversion:
+      converter: mrlEyes_2018_01
```

Comment:
I do not see the converter with this name

Reply:
Hmm, and now?